openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1401,8 +1401,10 @@ def check_vol_status(self):
                 for brick_or_shd in status[vol][host].keys():
                     if status[vol][host][brick_or_shd]['status'] != "1":
                         down_bricks += 1
-                    pid = status[vol][host][brick_or_shd]['pid']
-                    pids[pid] += 1
+
+                    if brick_or_shd != "Self-heal Daemon":
```

Comment:
It'll be nice if you also could add a check for "Snapshot Daemon" along with "Self-heal Daemon" if the volume is using snapshots.

Reply:
+1